src/index.js

```diff
@@ -0,0 +1,118 @@
+                                  <input type="checkbox" class="checkFunc" onclick="checkFunc(${element.index});" id="check${element.index}">
+                                  <input type="text" value="${element.description}" class="listitem" id="item${element.index}" readonly>
+                                  </div>
+                                  <div class = "tools">
+                                  <i class="fa-solid fa-pen-to-square edit " id="edit${element.index}" onclick="editFunc(${element.index});"></i>
+                                  <i class="fa-solid fa-floppy-disk save hide box" id="save${element.index}" onclick="saveFunc(${element.index});"></i>
+                                  <i id="removeicon" onclick="removeFunc(${element.index});" class="fa-solid fa-trash"></i>
+                                  </div>
+                          </li>`;
+    }
+  });
+
+  list.innerHTML = litItems;
+  add.value = '';
+};
+
+showList();
+// Add function
+add.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && add.value.length !== 0) {
+    const storedList = localStorage.getItem('todo');
+
+    if (storedList === null) {
+      todoList = [];
+    } else {
+      todoList = JSON.parse(storedList);
+      index = todoList.length === 0 ? 0 : todoList.length;
+    }
+
+    const LocalStore = {
+      index,
+      description: add.value,
+      completed,
+    };
+    todoList.push(LocalStore);
+    localStorage.setItem('todo', JSON.stringify(todoList));
+    showList();
+  }
+});
+
+// Call data from local storage
+window.onload = () => {
+  if (localStorage.getItem('todo')) {
+    todoList = JSON.parse(localStorage.getItem('todo'));
+  }
+  showList();
+};
+
+// edit todolist
+window.editFunc = (index) => {
+  const edit = document.getElementById(`edit${index}`);
+  const save = document.getElementById(`save${index}`);
+
+  save.style.display = 'flex';
+  edit.style.display = 'none';
+  const mainItem = document.getElementById(`item${index}`);
+  mainItem.removeAttribute('readonly');
+  const { length } = mainItem.value;
+  mainItem.setSelectionRange(length, length);
+  mainItem.focus();
+  return mainItem;
+};
+
+// Save function after editing
+window.saveFunc = (index) => {
+  const edit = document.getElementById(`edit${index}`);
+  const save = document.getElementById(`save${index}`);
+
+  save.style.display = 'none';
+  edit.style.display = 'flex';
+
+  const mainItem = document.getElementById(`item${index}`);
+  const storedData = localStorage.getItem('todo');
+  todoList = JSON.parse(storedData);
+  todoList[index].description = mainItem.value;
+
+  localStorage.setItem('todo', JSON.stringify(todoList));
+  showList();
+};
+
+// Remove function
+window.removeFunc = (index) => {
+  const storedData = localStorage.getItem('todo');
+  todoList = JSON.parse(storedData);
+  todoList.splice(index, 1);
+  for (let i = 0; i < todoList.length; i += 1) {
+    todoList[i].index = i;
+  }
```

Comment:
- Kindly, when you delete items it is resetting the index to start from `0` because you loop start from `0` and you are using it directly, so  please update it to start the index from 1 as the previous code reviewer suggested you :+1: 
you can start the loop variable from 1 or you can use i+1 in your index
```
for (let i = 1; i < todoList.length; i += 1) {
    todoList[i].index = i;
  }
```